### PR TITLE
fixes crowbars

### PR DESCRIPTION
**Pull request by AdamElTablawy**
check your fucking prs

### CODE REVIEW

**Comment by Rohesie** [comment]
This doesn't look like a bug.

**Comment by AdamElTablawy** [comment]
> This doesn't look like a bug.

Crowbars not fitting in toolbelts because someone made them too big to fit in toolbelts because they removed pocket crowbars for some inexplicable reason is not a bug, it is an oversight.

**Review by Rohesie — APPROVED** [review]
Yea, that much is true.
If they wanted to make them big they should have a toolbelt exception for it.

File: code/game/objects/items/tools/crowbar.dm [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	slot_flags = ITEM_SLOT_BELT
 	force = 15
 	throwforce = 7
-	w_class = WEIGHT_CLASS_NORMAL
+	w_class = WEIGHT_CLASS_TINY
```
Comment (Rohesie) [inline code/game/objects/items/tools/crowbar.dm:13]:
```suggestion
	w_class = WEIGHT_CLASS_SMALL
```
Don't these fit into belts?

**Pull request merged**